hivemind/dht/node.py

```diff
@@ -110,7 +114,7 @@ async def create(
         cache_refresh_before_expiry: float = 5,
         cache_on_store: bool = True,
         reuse_get_requests: bool = True,
-        num_workers: int = 1,
+        num_workers: int = DHT_DEFAULT_NUM_WORKERS,
```

Comment:
```suggestion
        num_workers: Optional[int] = None,
```

Related to [this](https://github.com/learning-at-home/hivemind/pull/342/files#r679582610).